### ISSUE TITLE
Update Windows path for installation in README

### ISSUE THREAD

**Issue by aentwist** [Klowner/inkscape-applytransforms#38]
I recently downloaded Inkscape 1.0.2 for Windows. I can confirm that the recommended path C:\Program Files\Inkscape\share\extensions does not work. The path is C:\Program Files\Inkscape\share\inkscape\extensions.

**Comment by vmario89** [comment]
you can also paste the extension in %appdata%/... user directory of inkscape btw. This will be update-safe in case you use portable version for example

**Comment by johnflux** [comment]
Yep, I ran into this issue too.  Created a pull request, with both of your comments: https://github.com/Klowner/inkscape-applytransforms/pull/39

**Comment by Klowner** [comment]
merged #39, thanks for the feedback and pull request.
